packages/test-app/frontend/package.json

```diff
@@ -10,7 +10,7 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.0.1",
     "@testing-library/user-event": "^13.5.0",
-    "@urturn/client": "^0.0.1-dev2",
+    "@urturn/client": "*",
```

Comment:
Note that this is how `lerna` lets us specify the other monorepo packages as dependencies that are versionless. We don't have to manage incrementing these versions everytime we release. `lerna` will take care of this for us, and uses this to know how to link dependent packages.